fabric-data-generation-api-v1/src/main/java/net/fabricmc/fabric/api/datagen/v1/JsonKeySortOrderAdder.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.datagen.v1;
+
+/**
+ * Provides a callback for setting the sort priority of object keys in generated JSON files.
+ */
+@FunctionalInterface
+public interface JsonKeySortOrderAdder {
```

Comment:
`Consumer` might be a bit of a better name, or `Callback`? We use `Builder` above, but I dont think thats correct in this case. `Adder` isnt really something we have used before anywhere.

Reply:
`Adder` is used in a couple of yarn class names for UI and was really the first thing I thought of, presumably because I was working with UI around that time. Anyhow, I'll change the name to `Callback`, as it's used in the javadoc anyways, so that's an additional consistency too.